src/profiler.cpp

```diff
@@ -66,9 +66,21 @@ void idfx::Profiler::Init() {
 void idfx::Profiler::Show() {
   idfx::pushRegion("Profiler::Show");
 
+  double usedMemory{-1};
+
+  // follow ISO/IEC 80000
+  const int NUNITS{5};
+  const char UNITS[NUNITS][100] {"B", "KB", "MB", "GB", "TB"};
```

Comment:
`const std::array<std::string,nUnits> {"B", "KB", "MB", "GB", "TB"};`
is more c++ like (and avoid the arbitrary size=100)